docusaurus/docs/operate/upgrades/3_module_auth.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Module Authorizations
+sidebar_position: 3
+---
+
+## Module Authorizations <!-- omit in toc -->
+
+Pocket Network utilizes an offchain governance mechanism that enables the community to vote on proposals.
+
+Once a proposal passes, or a decision by PNF is made on behalf of the DAO, PNF adjust the parameters necessary for the protocol's operation.
```

Comment:
Currently:
```suggestion
Once a proposal passes, or a decision by PNF is made on behalf of the DAO, the parameters updates are applied.
```

As soon as #1154 is merge, the following statement will be true:
```suggestion
Once a proposal passes, or a decision by PNF is made on behalf of the DAO, the parameter updates are pending activation, onchain, and active starting from the beginning of the next session.
```